sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -382,13 +382,14 @@ private[ui] class ExecutionPagedTable(
   }
 
   private def descriptionCell(execution: SQLExecutionUIData): Seq[Node] = {
+    val jobDescription = UIUtils.makeDescription(execution.description, basePath, plainText = false)
     val details = if (execution.details != null && execution.details.nonEmpty) {
```

Comment:
@srowen There is a much safer function. In terms of security, only anchor tags with root relative links are supported. So any attempts to embed links outside Spark UI, or other tags like "script" will cause in the whole description to be treated as plain text.